src/IndexGame.py

```diff
@@ -29,9 +28,46 @@
             }
 mapSize = [0,0]  #X-axis, Y-axis
 
+MainMenuData = \
+['Welcome, mayor of Simp City!',
+'------------------------------',
+'1. Start new game',
+'2. Load saved game',
+'3. Show high scores\n'
+'0. Exit']
+
 # General functions
-def displayMainMenu():
-    return True
+def displayMainMenu(): # To display the main menu
+    for i in range(len(MainMenuData)):
+            print(MainMenuData[i])
+   
+def runMainMenu(): # To run the main menu options
+    result = 1
+    while result != 0:
+        displayMainMenu()
+        userInput = input('Enter your option: ')
```

Comment:
Can we add some form of user input validation? You can tag on the pre-existing functions that have created

Reply:
The result is passed into result = MainMenuSelection(userInput) for validation